x-pack/plugins/upgrade_assistant/public/components/tabs/checkup/deprecations/reindex/flyout/checklist_step.tsx

```diff
@@ -53,40 +80,64 @@ export const ChecklistFlyoutStep: React.StatelessComponent<{
     <Fragment>
       <EuiFlyoutBody>
         <EuiCallOut
-          title="Index is unable to ingest, update, or delete documents while reindexing"
+          title={
+            <FormattedMessage
+              id="xpack.upgradeAssistant.checkupTab.reindexing.flyout.checklistStep.readonlyCallout.calloutTitle"
+              defaultMessage="Index is unable to ingest, update, or delete documents while reindexing"
+            />
+          }
           color="warning"
           iconType="alert"
         >
           <p>
-            If you can’t stop document updates or need to reindex into a new cluster, consider using
-            a different upgrade strategy.
+            <FormattedMessage
+              id="xpack.upgradeAssistant.checkupTab.reindexing.flyout.checklistStep.readonlyCallout.cantStopDetail"
+              defaultMessage="If you can’t stop document updates or need to reindex into a new cluster,
+                consider using a different upgrade strategy."
+            />
           </p>
           <p>
-            Reindexing will continue in the background, but if Kibana shuts down or restarts you
-            will need to return to this page to resume reindexing.
+            <FormattedMessage
+              id="xpack.upgradeAssistant.checkupTab.reindexing.flyout.checklistStep.readonlyCallout.backgroundResumeDetail"
+              defaultMessage="Reindexing will continue in the background, but if Kibana shuts down or restarts you will
+                need to return to this page to resume reindexing."
+            />
           </p>
         </EuiCallOut>
         {!hasRequiredPrivileges && (
           <Fragment>
             <EuiSpacer />
             <EuiCallOut
-              title="You do not have sufficient privileges to reindex this index"
+              title={
+                <FormattedMessage
+                  id="xpack.upgradeAssistant.checkupTab.reindexing.flyout.checklistStep.insufficientPrivilegeWarningCallout.title"
+                  defaultMessage="You do not have sufficient privileges to reindex this index"
+                />
+              }
               color="danger"
               iconType="alert"
             />
           </Fragment>
         )}
         <EuiSpacer />
         <EuiTitle size="xs">
-          <h3>Reindexing process</h3>
+          <h3>
+            <FormattedMessage
+              id="xpack.upgradeAssistant.checkupTab.reindexing.flyout.checklistStep.reindexingChecklistTitle"
```

Comment:
Nit: in `xpack.upgradeAssistant.checkupTab.reindexing.flyout.checklistStep.insufficientPrivilegeWarningCallout.title` you broke out the title, where here it's included in the top level key: `xpack.upgradeAssistant.checkupTab.reindexing.flyout.checklistStep.reindexingChecklistTitle`

Reply:
I intended the callouts to be consistent, though it wasn't quite consistent with that either.

Each callout should have its own identifier, with the penultimate suffix being individual parts of the callout. For example, `myCallout.calloutTitle` and `myCallout.calloutDetail`.


I'm going to change `xpack.upgradeAssistant.checkupTab.reindexing.flyout.checklistStep.insufficientPrivilegeWarningCallout.title` to `xpack.upgradeAssistant.checkupTab.reindexing.flyout.checklistStep.insufficientPrivilegeCallout.calloutTitle` and leave this one alone.